From 8de6f25db1afc993fda862a886b59fb3cc29f5b9 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Wed, 16 Oct 2019 10:34:15 +0800
Subject: [PATCH] arm64: rockchip_defconfig: remove nonexistent config

-CONFIG_DEFAULT_USE_ENERGY_AWARE
-CONFIG_MEMORY_STATE_TIME
-CONFIG_GPIO_RK8XX
-CONFIG_MMC_BLOCK_BOUNCE
-CONFIG_ANDROID_TIMED_GPIO
-CONFIG_ANDROID_LOW_MEMORY_KILLER
-CONFIG_SYNC
-CONFIG_SW_SYNC_USER
-CONFIG_ION_ROCKCHIP
-CONFIG_TIMER_STATS
-CONFIG_DEBUG_SET_MODULE_RONX

Change-Id: I2bf328e70a54878e28084becb9de1f791cc12b1e
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 21 ++++-----------------
 1 file changed, 4 insertions(+), 17 deletions(-)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index c285846e10b5..6b78717a656b 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -23,7 +23,6 @@ CONFIG_NAMESPACES=y
 # CONFIG_UTS_NS is not set
 # CONFIG_PID_NS is not set
 CONFIG_SCHED_TUNE=y
-CONFIG_DEFAULT_USE_ENERGY_AWARE=y
 CONFIG_RELAY=y
 CONFIG_BLK_DEV_INITRD=y
 # CONFIG_RD_BZIP2 is not set
@@ -249,7 +248,7 @@ CONFIG_BT_HCIUART_H4=y
 CONFIG_BT_HCIUART_LL=y
 CONFIG_RFKILL=y
 CONFIG_DEVTMPFS=y
-CONFIG_FW_LOADER_USER_HELPER_FALLBACK=y
+CONFIG_FW_LOADER_USER_HELPER=y
 CONFIG_MALI_MEMORY_GROUP_MANAGER=y
 CONFIG_DMA_CMA=y
 CONFIG_ZRAM=y
@@ -260,7 +259,6 @@ CONFIG_BLK_DEV_NVME=y
 CONFIG_ROCKCHIP_SCR=y
 CONFIG_SRAM=y
 CONFIG_UID_SYS_STATS=y
-CONFIG_MEMORY_STATE_TIME=y
 CONFIG_USB_CAM_GPIO=y
 CONFIG_SCSI=y
 CONFIG_BLK_DEV_SD=y
@@ -489,7 +487,9 @@ CONFIG_DEBUG_GPIO=y
 CONFIG_GPIO_SYSFS=y
 CONFIG_POWER_AVS=y
 CONFIG_ROCKCHIP_IODOMAIN=y
-CONFIG_GPIO_RK8XX=y
+CONFIG_POWER_RESET_GPIO=y
+CONFIG_SYSCON_REBOOT_MODE=y
+CONFIG_TEST_POWER=y
 CONFIG_CHARGER_BQ25700=y
 CONFIG_BATTERY_EC=y
 CONFIG_BATTERY_CW2015=y
@@ -497,9 +497,6 @@ CONFIG_BATTERY_RK817=y
 CONFIG_CHARGER_RK817=y
 CONFIG_BATTERY_RK818=y
 CONFIG_CHARGER_RK818=y
-CONFIG_POWER_RESET_GPIO=y
-CONFIG_SYSCON_REBOOT_MODE=y
-CONFIG_TEST_POWER=y
 CONFIG_THERMAL=y
 CONFIG_THERMAL_WRITABLE_TRIPS=y
 CONFIG_THERMAL_DEFAULT_GOV_POWER_ALLOCATOR=y
@@ -745,14 +742,12 @@ CONFIG_USB20_HOST=y
 CONFIG_USB20_OTG=y
 CONFIG_MMC=y
 CONFIG_MMC_BLOCK_MINORS=32
-# CONFIG_MMC_BLOCK_BOUNCE is not set
 CONFIG_MMC_SDHCI=y
 CONFIG_MMC_SDHCI_PLTFM=y
 CONFIG_MMC_SDHCI_OF_ARASAN=y
 CONFIG_MMC_DW=y
 CONFIG_MMC_DW_ROCKCHIP=y
 CONFIG_LEDS_GPIO=y
-CONFIG_LEDS_TRIGGERS=y
 CONFIG_LEDS_TRIGGER_BACKLIGHT=y
 CONFIG_LEDS_TRIGGER_DEFAULT_ON=y
 CONFIG_SWITCH=y
@@ -765,13 +760,8 @@ CONFIG_SW_SYNC=y
 CONFIG_STAGING=y
 CONFIG_INV_MPU_IIO=y
 CONFIG_ASHMEM=y
-CONFIG_ANDROID_TIMED_GPIO=y
-CONFIG_ANDROID_LOW_MEMORY_KILLER=y
-CONFIG_SYNC=y
-CONFIG_SW_SYNC_USER=y
 CONFIG_ION=y
 CONFIG_ION_SYSTEM_HEAP=y
-CONFIG_ION_ROCKCHIP=y
 CONFIG_FIQ_DEBUGGER=y
 CONFIG_FIQ_DEBUGGER_NO_SLEEP=y
 CONFIG_FIQ_DEBUGGER_CONSOLE=y
@@ -822,7 +812,6 @@ CONFIG_PHY_ROCKCHIP_TYPEC=y
 CONFIG_PHY_ROCKCHIP_USB=y
 CONFIG_ANDROID=y
 CONFIG_ANDROID_BINDER_IPC=y
-CONFIG_NVMEM=y
 CONFIG_ROCKCHIP_EFUSE=y
 CONFIG_ROCKCHIP_OTP=y
 CONFIG_TEE=y
@@ -860,8 +849,6 @@ CONFIG_NLS_CODEPAGE_437=y
 CONFIG_NLS_ASCII=y
 CONFIG_NLS_ISO8859_1=y
 CONFIG_NLS_UTF8=y
-CONFIG_TIMER_STATS=y
-CONFIG_DEBUG_SET_MODULE_RONX=y
 CONFIG_SECURITY_PERF_EVENTS_RESTRICT=y
 CONFIG_SECURITY=y
 CONFIG_SECURITY_NETWORK=y
-- 
2.35.3

